site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -62,11 +62,31 @@ The type of security protocol FusionAuth will use when connecting to the SMTP se
 [field]#tenant.emailConfiguration.setPasswordEmailTemplateId# [type]#[UUID]# [optional]#Optional#::
 The Id of the Email Template that is used when a user had their account created for them and they must set their password manually and they are sent an email to set their password.
 
+[field]#tenant.emailConfiguration.unverified.allowEmailChangeWhenGated# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.27.0#::
+When this value is set to `true`, the user is allowed to change their email address when gated.
```

Comment:
```suggestion
When this value is set to `true`, the user is allowed to change their email address when they are gated because they haven't verified their email address.
```